lib/Madara.lua

```diff
@@ -195,7 +195,7 @@ function defaults:parseNovel(url, loadChapters)
 										:add("manga", id):build())
 				)
 			else
-				-- Used by BoxNovel, NovelTrench and LightNovelHeaven.
+				-- Used by BoxNovel, NovelTrench, LightNovelHeaven and VipNovel.
```

Comment:
This comment is be starting to get bloated and I believe more and more sites will start using this so I would remove it. Those who feel the need to know which extensions are affected can `grep` through the extensions.